src/core/server/elasticsearch/elasticsearch_config.ts

```diff
@@ -33,7 +33,7 @@ export const configSchema = schema.object({
   }),
   sniffOnConnectionFault: schema.boolean({ defaultValue: false }),
   hosts: schema.oneOf([hostURISchema, schema.arrayOf(hostURISchema, { minSize: 1 })], {
-    defaultValue: 'http://localhost:9200',
+    defaultValue: 'https://localhost:9200',
```

Comment:
__question:__ do we have a good reason to change this? Without proper TLS configuration (`elasticsearch.ssl.*`) it wouldn't work out-of-the-box anyway, but if we keep `http` we can use it as a flag that user didn't try to set up TLS on their own and we can activate interactive setup mode.